src/index.js

```diff
@@ -57,7 +57,7 @@ export default async function microbundle(inputOptions) {
 	options.name = finalName;
 	options.pkg.name = pkgName;
 
-	if (options.sourcemap !== false) {
+	if (options.sourcemap !== false && options.sourcemap !== 'inline') {
 		options.sourcemap = true;
 	}
```

Comment:
Might be good to have a warning for this, so folks don't use the option thinking it's a way to make output cleaner (fewer files).

```suggestion
	if (options.sourcemap === 'inline') {
		console.log(`Warning: inline sourcemaps should only be used for debugging purposes.`);
		options.sourcemap = true;
	} else if (options.sourcemap !== false) {
		options.sourcemap = true;
	}
```

Reply:
Good idea 👍🏻